app/services/rbac_api.rb

```diff
@@ -1,32 +1,47 @@
 # frozen_string_literal: true
 
-# This class is meant to handle calls to the RBAC light API
+require 'insights-rbac-api-client'
+
+# This class is meant to handle calls to the RBAC API
 class RbacApi
+  COMPLIANCE_FULL_ACCESS = 'compliance:*:*'
+  APPS = 'compliance,inventory'
+
   def initialize(b64_identity)
-    @url = URI.parse("#{URI.parse(Settings.rbac_url)}"\
-                     "#{Settings.path_prefix}/rbac/v1/access/")
+    RBACApiClient.configure do |config|
+      config.host = Settings.rbac_url
+      config.username = Platform::BASIC_AUTH[0] # FIXME: Settings.rbac_username
+      config.password = Platform::BASIC_AUTH[1] # FIXME: Settings.rbac_password
+    end
```

Comment:
This might be something we could extract into an initializer under `config/initializers`